osquery/logger/plugins/aws_util.h

```diff
@@ -0,0 +1,121 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <memory>
+
+#include <aws/core/Region.h>
+#include <aws/core/auth/AWSCredentialsProvider.h>
+#include <aws/core/auth/AWSCredentialsProviderChain.h>
+#include <aws/core/client/ClientConfiguration.h>
+#include <aws/core/http/HttpClient.h>
+#include <aws/core/http/HttpClientFactory.h>
+#include <aws/core/http/standard/StandardHttpResponse.h>
+
+#include <boost/property_tree/ini_parser.hpp>
```

Comment:
Ok, let's just remove the include from this header since from what I can tell it's only needed in the implementation.

We'll see about removing it for manual parsing later on.